inst/examples/06-publishing.Rmd

```diff
@@ -19,42 +19,103 @@ bookdown::publish_book(render = 'local')
 
 If you have set up your own RStudio Connect server, you can certainly publish the book to that server instead of bookdown.org.
 
-## GitHub
+## GitHub Pages {#github}
 
-You can host your book on GitHub\index{GitHub} for free via GitHub Pages (https://pages.github.com). GitHub supports Jekyll (http://jekyllrb.com), a static website builder, to build a website from Markdown files. That may be the more common use case of GitHub Pages, but GitHub also supports arbitrary static HTML files, so you can just host the HTML output files of your book on GitHub. The key is to create a hidden file `.nojekyll` that tells GitHub that your website is not to be built via Jekyll, since the **bookdown** HTML output is already a standalone website. 
+You can host your book on GitHub\index{GitHub} for free via GitHub Pages (<https://pages.github.com>).^[See the [GitHub Pages Help documents](http://bit.ly/2cvloKV) for more information.] The simplest approach is to publish your book as a [GitHub Project Page](https://help.github.com/en/articles/user-organization-and-project-pages#project-pages-sites) from a `/docs` folder on your `main` branch, which will then be available at `https://<USER>.github.io/<REPO>` (unless you provide a custom domain name). The main benefit to this approach is its simplicity, and the fact that you can track the source files for your book and the published HTML files in the same branch.
```

Comment:
Is this the preferred way all the time compared to `gh-pages` ? 
Or is this what you think is simpler to use with Bookdown ? 

My view on this: 
* `/docs` folder is simpler when not using a CI: 
   * Build locally to `docs/`
   * Commit and push

* I find `gh-pages` cleaner to have in the source repo and easier to put in place when using a CI. Locally I use a Makefile to help me generally 
  * On main branch, `_book` (or other output dir) is in `.gitignore`
  * I build the book locally 
  * Then I switch branch to gh-pages
  * I commit the `_book` folder
  * I push the gh-pages branch 
  * I switched back to main branch

Makefile do all that for me usually. Using a R file with **gert** should use the same. An R function could help with that.

Anyway, just sharing how I work with this.

Reply:
I would definitely suggest the `gh-pages` route for folks who are comfortable with Make, gert, and CI systems like GHA (like yourself). I can draft that section, but could use a draft GHA workflow (much easier than Make/gert) since there is no `deploy_to_branch()` :)

Reply:
Ok so we agree on this. 

I suggest we do the GHA part later - this would require to rework a bit a workflow, and I would like to show netlify and rstudio connect for all options. Best would be to have demo repo too with all this !  
So it is a bit of work to it right IMO. 

It is also possible to work on helper function to help deploy. We have already `publish_book()` for rstudio connect, we could have another one for GH Pages deployement (but it could also be directly in the Workflow template). 

I can work ont that... but later in another PR. 

How does that sounds ?